cmd/firehose-tendermint/app_firehose.go

```diff
@@ -82,7 +84,26 @@ func init() {
 			grcpShutdownGracePeriod = shutdownSignalDelay - (5 * time.Second)
 		}
 
+		indexStoreUrl := viper.GetString("firehose-block-index-url")
```

Comment:
This CLI arg must be defined (see `registerFlags` part), otherwise you'll get error: 

```
Error: cannot launch un-registered application "firehose-block-index-url=foo"
```

Reply:
I fixed this